### DISCUSSION TITLE
Vouchers/gift cards?

### DISCUSSION THREAD

**Discussion by Glenf** [duncanmcclean/simple-commerce#863] [Feature Requests]
Hi,

I'm planning on adding Simple Commerce to a site. I was wondering is there a possibility to have (digital/printable) vouchers or gift cards?

 I saw that digital products addon has license key possibility, but it's not probably what I want.

**Comment by duncanmcclean** [comment]
This functionality isn't available in Simple Commerce, no. In the aim of keeping Simple Commerce _simple_, it's not something I'm planning to add either.

The closest Simple Commerce has is the concept of coupons.

However, it's perfectly possible for you to build this feature yourself (& make an addon out of it for others who may also want this functionality).

**Reply by Glenf** [reply]
Thank you for a quick reply. I might have to take a try on creating this addon.
